SPECS/softhsm/softhsm.spec

```diff
@@ -32,28 +27,27 @@ designed to meet the requirements of OpenDNSSEC, but can also work together
 with other cryptographic products because of the PKCS#11 interface.
 
 %package devel
-Summary: Development package of softhsm that includes the header files
-Requires: %{name} = %{version}-%{release}, openssl-devel, sqlite-devel
-%if 0%{?prever:1} || 0%{?prerelease:1}
-BuildRequires: autoconf, libtool, automake
-%endif
+Summary:        Development package of softhsm that includes the header files
+BuildRequires:  autoconf
+BuildRequires:  automake
+BuildRequires:  libtool
+Requires:       %{name} = %{version}-%{release}
+Requires:       openssl-devel
+Requires:       sqlite-devel
 
 %description devel
 The devel package contains the libsofthsm include files
 
 %prep
-%setup -q -n %{name}-%{version}%{?prever}
+%setup -q -n %{name}-%{version}
 %patch1 -p1
 
-%if 0%{?prever:1} || 0%{?prerelease:1}
-   # pre-release or post-release snapshots fixup
-   sed -i 's:^full_libdir=":#full_libdir=":g' configure.ac
-   sed -i "s:libdir)/@PACKAGE@:libdir):" Makefile.in
+# pre-release or post-release snapshots fixup
+sed -i 's:^full_libdir=":#full_libdir=":g' configure.ac
+sed -i "s:libdir)/@PACKAGE@:libdir):" Makefile.in
 autoreconf -fiv
```

Comment:
Similar comment about removing `prever` and `prerelease`: we should only take one of the branches of the `%if` statement and in this case it's going to be the `%else` part, since none of the macros is defined:

```suggestion
```